test/lit.cfg

```diff
@@ -733,6 +733,65 @@ elif run_os == 'linux-gnu' or run_os == 'linux-gnueabihf' or run_os == 'freebsd'
     config.target_ld = (
         "ld -L%s" %
         (os.path.join(test_resource_dir, config.target_sdk_name)))
+elif run_os == 'linux-androideabi':
+    # Android
+    lit_config.note("Testing Android " + config.variant_triple)
+    # FIXME: This value should be read off of config.android_ndk_path,
+    #        but I can't figure out how. The value is set via CMake, and the
+    #        build directory contains a test-android-armv7/lit.site.cfg with
+    #        its @SWIFT_ANDROID_NDK_PATH@ correctly replaced. So what's wrong?
```

Comment:
That was it!! Thanks for the tip. Yes, chaining would be nice. I think this would make a good starter task, so I wrote up a Swift report: https://bugs.swift.org/browse/SR-1266.